testbed/testbed/senders.go

```diff
@@ -114,6 +119,10 @@ func (dsb *DataSenderBase) Flush() {
 	// Exporter interface does not support Flush, so nothing to do.
 }
 
+func (dsb *DataSenderBase) GetNetworkProtocol() string {
```

Comment:
Is it not better to change `GetEndpoint` to return `net.IpAddr`?

Reply:
need use testbed test a udp server, use `net.Ipaddr` no way to distinguish between tcp and udp

Reply:
Sorry, I meant `net.Addr` which has a network value which can be tcp/udp/etc.

Reply:
good idea, let me change it :)